Dockerfile.musl

```
@@ -101,7 +101,7 @@ ARG DEFAULT_CFLAGS
 ARG USE_MIMALLOC
 ARG USE_EXTERNAL_MIMALLOC
 
-RUN apk add --no-cache cpio curl file gnupg ninja ruby ruby-getoptlong unzip rsync perl python3 openssl-dev openssl linux-headers
+RUN apk add --no-cache cpio curl file gnupg ninja nodejs ruby ruby-getoptlong unzip rsync perl python3 openssl-dev openssl linux-headers
```

Hadolint (2.15.1)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)